rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -67,11 +67,11 @@ http {
           balancer = res
         end
 
-        ok, res = pcall(require, "monitor")
+        ok, res = pcall(require, "statsd.monitor")
```

Comment:
can you rename it to `statsd_monitor.lua`?

Reply:
👍 from what I've seen "statsd.monitor" may mean that the file is  `statsd/monitor.lua`. Could top hat to verify that there's no problem requiring the module - but renaming to `statsd_monitor.lua` should fix that concern.